lib/mongo/bulk_write/combineable.rb

```diff
@@ -28,6 +28,10 @@ module Combineable
       # @return [ true, false ] has_collation Whether one or more operations has a collation defined.
       attr_reader :has_collation
 
+      #return [true, false] has_array_filters Whether one or more operations specifies an array
```

Comment:
Format should be ``# @return`` so that yard documentation can pick up the ``@return`` tag properly.
Also, we usually put a space before/ after the ``[`` and ``]``